index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Tarikwa Portfolio</title>
+</head>
+
+<body>
+    <section class="main-section">
+        <section class="toolbar">
+            <ul class="nav">
+                <li class="item"><a href="#">My Logo</a></li>
+                <li class="item"><a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a></li>
+            </ul>
+        </section>
+        <section class="headline-section">
+            <h1>I'm Tarikwa <br> Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] I would also like to suggest that you use the font color `#344563` for the description text instead of `black`. This way, you would be following the design template provided by Figma. 

## Your design
![image](https://user-images.githubusercontent.com/77106962/143409941-717d0891-eb9c-47da-ac57-3290eb35ee95.png)

## Figma's Design
![image](https://user-images.githubusercontent.com/77106962/143410051-63ff66d2-1eac-4b3d-a9bd-ab687531d78c.png)

Reply:
I have fixed this by changing the color.